iothub_client/tests/common_longhaul/iothub_client_common_longhaul.c

```diff
@@ -1417,7 +1404,8 @@ static void on_c2d_message_sent(void* context, IOTHUB_MESSAGING_RESULT messaging
         C2D_MESSAGE_INFO info;
         info.message_id = send_context->message_id;
         info.send_callback_result = messagingResult;
-        info.time_sent = time(NULL);
+        info.time_sent = get_time(NULL);
+        LogInfo("on_c2d_message_sent(#%d) messagingResult=%s, %s", send_context->message_id, MU_ENUM_TO_STRING(IOTHUB_MESSAGING_RESULT, messagingResult), get_ctime(&info.time_sent));
```

Comment:
Nit - do we need the `get_ctime` call here?  Looking at the log runs for both Win+Linux it appears as if LogInfo is setup such that the time entry is automatically appended anyway, and it'd be (almost) the same since the info_time_sent is specified on the line immediately above.

Reply:
no, only LogError pre-appends a time stamp to the message. We need to do this here do that we can track when the hub received the C2D message.